deploy/summon/secrets.yml

```
@@ -41,17 +41,45 @@ oc311:
   PULL_DOCKER_REGISTRY_URL: !var ci/openshift/3.11/registry-url
   PULL_DOCKER_REGISTRY_PATH: !var ci/openshift/3.11/registry-url
 
-oc45:
-  OPENSHIFT_VERSION: '4.5'
+oldest:
+  OPENSHIFT_VERSION: !var ci/openshift/oldest/version
+  OPENSHIFT_URL: !var ci/openshift/oldest/hostname
+  OPENSHIFT_USERNAME: !var ci/openshift/oldest/username
+  OPENSHIFT_PASSWORD: !var ci/openshift/oldest/password
+
+  PLATFORM: openshift
+  TEST_PLATFORM: !var ci/openshift/oldest/version
```

it looks like kubernetes-conjur-deploy expects this to be `openshift*` (though I'm not sure why it's working now)
https://github.com/cyberark/kubernetes-conjur-deploy/blob/fbe14df8a20e451ffdebca0d4b73c85df52d2bcc/test.sh#L43-L51

you can probably just make them all the literal strings `openshift` instead
```suggestion
  TEST_PLATFORM: openshift
```

---

btw, this seems silly and redundant now. makes me think we should just change the logic in k8s-deploy to use `PLATFORM`, remove `TEST_PLATFORM` from that project, and then remove it from here.